management/server/user.go

```diff
@@ -189,6 +193,61 @@ func (am *DefaultAccountManager) CreateUser(accountID, userID string, invite *Us
 
 }
 
+// AddPATToUser takes the userID and the accountID the user belongs to and assigns a provided PersonalAccessToken to that user
+func (am *DefaultAccountManager) AddPATToUser(accountID string, userID string, pat *PersonalAccessToken) error {
+	unlock := am.Store.AcquireAccountLock(accountID)
+	defer unlock()
+
+	account, err := am.Store.GetAccount(accountID)
+	if err != nil {
+		return err
+	}
+
+	user := account.Users[userID]
+	if user == nil {
+		return status.Errorf(status.NotFound, "user not found")
+	}
+
+	user.PATs[pat.ID] = pat
+
+	err = am.Store.SaveAccount(account)
```

Comment:
`return am.Store.SaveAccount(account)`

Reply:
I kept that in two lines to improve readability because return type of SaveAccount wouldn't be intuitive to me (would have to check func definition) but I can do single line